### DISCUSSION TITLE
MOES 3 Gang Smart Switch Module

### DISCUSSION THREAD

**Discussion by 4zoom** [arendst/Tasmota#11351] [Q&A] [answered]
Hey guys

I got a new device "MOES 3 Gang Smart Switch Module". I haven't found so far no documentation how to flash it with tasmota

Has someone seen it before and have some insides about it?

![IMG_20210315_094636 (2021-03-15T09_09_56 846)](https://user-images.githubusercontent.com/78219551/111129971-fcde6300-8576-11eb-9797-88e35f0ac25d.jpg)
![IMG_20210315_094649 (2021-03-15T09_10_32 043)](https://user-images.githubusercontent.com/78219551/111129988-01a31700-8577-11eb-840d-2ce0c2b92985.jpg)

**Comment by barbudor** [comment, marked as answer]
Hi
You can forget about Tasmota on this Wb3s module. This is not an ESP

Only solution is unsolder it and replace by an ESP module

**Reply by barbudor** [reply]
Details on Wb3s here https://developer.tuya.com/en/docs/iot/device-development/module/wifibt-dual-mode-module/wb-series-module/wb3s-module-datasheet?id=K9dx20n6hz5n4

Esp12F should be a good candidate

**Reply by 4zoom** [reply]
Thank you for the quick response !!!
